caikit/runtime/http_server/http_server.py

```diff
@@ -112,6 +112,7 @@
 # Endpoint to use for server info
 RUNTIME_INFO_ENDPOINT = "/info/version"
 
+
```

Comment:
I've seen a number of these additional newlines. I'm surprised our previous `black` formatter didn't catch these OR that it's not complaining about them now. I care zero about how it looks as long as `black` is happy though